Sources/Smithy/ByteStream.swift

```diff
@@ -8,7 +8,7 @@
 import class Foundation.FileHandle
 import struct Foundation.Data
 
-public enum ByteStream {
+public enum ByteStream: Sendable {
```

Comment:
This conformance is possible since `Stream` has been made `Sendable` in this PR.